section1/exercises/strings.rb

```diff
@@ -7,9 +7,11 @@
 p "Alan Turing"
 
 # Write code that prints `Welcome to Turing!` to the terminal:
-p #YOUR CODE HERE
+p "'Welcome to Turing!'"
```

Comment:
I notice you've got double quotes around single quotes here and below. That shows extra attention to details making sure that it prints the quotes as well as the string like in the instructions! I'm not 100% sure it was necessary but I'm super impressed you picked up on that.

Reply:
Thanks! It made sense to me because that's how I would quote someone quoting someone else (ex: Bob said, "Mary said, 'my favorite season is Summer.'"